package.json

```diff
@@ -54,16 +54,16 @@
     "@covalent/markdown": "1.0.0-beta.3",
     "@swimlane/ngx-charts": "^5.0.0",
     "core-js": "^2.4.1",
+    "d3": "^4.6.0",
     "hammerjs": "^2.0.8",
     "highlight.js": "9.10.0",
     "rxjs": "^5.2.0",
     "showdown": "1.6.4",
-    "zone.js": "^0.8.5",
-    "d3": "^4.6.0"
+    "zone.js": "^0.8.5"
   },
   "devDependencies": {
     "@angular/cli": "1.0.0",
-    "@angular/compiler-cli": "4.0.0",
+    "@angular/compiler-cli": "4.0.2",
```

Comment:
Can you change this to `^4.0.0`? So its the same as the rest of the `@angular` dependencies. 😄